headers/functions/arm9.h

```diff
@@ -48,6 +48,8 @@ enum move_category GetMoveCategory(enum move_id move_id);
 void HandleSir0Translation(uint8_t** dst, uint8_t* src);
 void HandleSir0TranslationVeneer(uint8_t** dst, uint8_t* src);
 int GetLanguage(void);
+int PreprocessString(char* output, int output_size, char* format, 
+                     struct preprocessor_flags flags, struct preprocessor_args* args);
```

Comment:
[Formatter caught](https://github.com/UsernameFodder/pmdsky-debug/runs/5298100158?check_suite_focus=true) that there's enough room to put `struct preprocessor_flags flags` on the previous line.